DynamoDB/CreateTable.js

```diff
@@ -44,7 +44,7 @@ var params = {
         }
     }],
     LocalSecondaryIndexes: [{
-        IndexName: "FILTER_BY_NAME",
+        IndexName: "FIND_FILE_BY_NAME",
         KeySchema: [
             {
                 AttributeName: "PK",
```

Comment:
For LSI we need to define only SK why have you define PK?

Reply:
Yes we need to mention both PK as well as SK while defining LSI "FIND_FILE_BY_NAME"

Reply:
so where is SK?

Reply:
I think you are missing something.
"LS1_SK" is defined as the Sort Key for LSI "FIND_FILE_BY_NAME".